build/Containerfile.controlplane

```diff
@@ -1,16 +1,29 @@
 FROM --platform=$BUILDPLATFORM tonistiigi/xx AS xx-tools
 
-FROM --platform=$BUILDPLATFORM rust:alpine
-ARG TARGETPLATFORM
-ARG PROJECT_DIR=/workspace
-ARG BUILD_DIR=$PROJECT_DIR/build
+FROM --platform=$BUILDPLATFORM rust:alpine AS builder
 
 RUN apk add --no-cache clang lld
-COPY --from=xx-tools / /
 
 WORKDIR /workspace
+
+COPY --from=xx-tools / /
+
+ARG TARGETPLATFORM
+ARG PROJECT_DIR=/workspace
+ARG BUILD_DIR=$PROJECT_DIR/build
+
 RUN --mount=type=bind,source=../controlplane/src/,target=src \
     --mount=type=bind,source=../controlplane/Cargo.toml,target=Cargo.toml \
     --mount=type=bind,source=../controlplane/Cargo.lock,target=Cargo.lock \
     xx-cargo build --release --target-dir $BUILD_DIR && \
     xx-verify ./build/$(xx-cargo --print-target-triple)/release/controller
+
+RUN cp ./build/$(xx-cargo --print-target-triple)/release/controller /workspace/manager
+
+FROM alpine:latest
+
+WORKDIR /
+
+COPY --from=builder /workspace/manager /manager
+
+RUN chmod +x /manager
```

Comment:
Maybe we don't need this line?

Reply:
we do not.

Reply:
sorry for the late reply,been travelling to attend Kubecon India 😅,and since we have used ./manger to start the controller,not sure why we dont need this line

Reply:
Because the manager binary is already executable